feat: update readme

viralgupta opened the pull request:
closes #16

viralgupta committed:
feat: update readme

viralgupta committed:
Update Node Neutralino Logo

shalithasuranga commented on README.md:
````diff
@@ -1,47 +1,181 @@
+> 
+> - Add config in `neutralino.config.json` for projectRunner
+> ```json
+> // This is required since node-neutralino communicates via neutralinojs extension protocol.
````
Could you please use `js` for code highlighting. Now, GitHub renders some lines in red since they are not valid JSON statements. Thanks

shalithasuranga commented on README.md:
```diff
@@ -1,47 +1,181 @@
+- This is a guide to show you how you can create similar package like `node-neutralino` in any language, eg: `py-neutralino`, `rust-neutralino`, etc.
+- Steps:
+- 1st Step (Spawning Neutralino Binaries With Process Args):
+  - Take input from users for all the [supported args](https://github.com/neutralinojs-community/node-neutralino/blob/main/types/api/window.ts).
```
This link is broken. Please check

shalithasuranga reviewed (changes requested):
Hello @viralgupta, thanks so much for sending this PR. Please check the inline notes and make sure to improve the markdown formatting by following [this guide](https://google.github.io/styleguide/docguide/style.html) Thanks :tada:

viralgupta committed:
Merge branch 'main' of https://github.com/neutralinojs-community/node-neutralino into feat/update-readme

viralgupta committed:
Merge branch 'feat/update-readme' of https://github.com/viralgupta/node-neutralino into feat/update-readme

viralgupta committed:
chore: fixed broken link

viralgupta committed:
feat: added TOC and updated formattting

viralgupta commented:
@shalithasuranga thank you sir for reviewing the PR, I have updated the PR with the requested changes, added TOC, and fixed some formatting issues.

Please let me know if there are any specify formatting issues I missed. 😇

shalithasuranga reviewed (approved):
Thanks for sending these updates. Looks great now :tada:
